diff --git a/src/script/nix-prefetch-git b/src/script/nix-prefetch-git
index 3fe399c8..fd10c18a 100755
--- a/src/script/nix-prefetch-git
+++ b/src/script/nix-prefetch-git
@@ -136,11 +136,11 @@ checkout_ref(){
 # Update submodules
 init_submodules(){
     # Add urls into .git/config file
-    git submodule init
+    git submodule init || echo "WARNING: git submodule init failed" >&2
 
     # list submodule directories and their hashes
-    git submodule update
-    git submodule status
+    git submodule update || echo "WARNING: git submodule update failed" >&2
+    git submodule status || echo "WARNING: git submodule status failed" >&2
 }
 
 clone(){
